web/packages/selfhosted/test/polyfill/object_unexpected_string/test.ts

```diff
@@ -23,7 +25,7 @@ describe("Object with unexpected string", () => {
     it("Plays a movie", async () => {
         await playAndMonitor(
             browser,
-            await browser.$("#test-container").$("<ruffle-embed />"),
+            await browser.$("#test-container").$("ruffle-embed"),
```

Comment:
Would these changes already work on wdio 8.x, @danielhjacobs?
Just to reduce the outstanding diff...

Reply:
I believe so and I actually didn't see a note that this is necessary with version 9 but it seems to be because the bidi selectors are the same as css selectors.

Reply:
These changes are no longer needed.